setParameters.m

```diff
@@ -91,13 +100,13 @@
     cfg.target.maxNbPerBlock = 2;
     cfg.target.duration = 0.05; % In secs
 
-    cfg.extraColumns = {'direction', 'speed', 'target', 'event', 'block'};
-    
+    cfg.extraColumns = {'direction', 'speed', 'target', 'event', 'block', 'keyName'};
```

Comment:
for extra precautions we wanto also collect the key pressed by the subject to distinguish keypresses from trigger